Shared/Samples/Manage operational layers/ManageOperationalLayersView.swift

```diff
@@ -0,0 +1,155 @@
+                    try await elevationImageLayer.load()
+                    
+                    let censusTiledLayer = ArcGISMapImageLayer(url: .censusTiles)
+                    try await censusTiledLayer.load()
+                    
+                    map.addOperationalLayers([elevationImageLayer, censusTiledLayer])
+                } catch {
+                    self.error = error
+                }
+            }
+            .toolbar {
+                ToolbarItem(placement: .bottomBar) {
+                    Button("Manage Layers") {
+                        isShowingSheet = true
+                    }
+                    .sheet(isPresented: $isShowingSheet, detents: [.medium], dragIndicatorVisibility: .visible) {
+                        NavigationView {
+                            ManageLayersSheetView(map: map)
+                                .navigationTitle("Manage Layers")
+                                .navigationBarTitleDisplayMode(.inline)
+                        }
+                    }
+                }
+            }
+            .alert(isPresented: $isShowingAlert, presentingError: error)
+    }
+}
+
+struct ManageLayersSheetView: View {
+    /// The map with the operational layers.
+    let map: Map
+    
+    /// The action to dismiss the manage layers sheet.
+    @Environment(\.dismiss) private var dismiss
+    
+    /// An array for all the layers currently on the map.
+    @State private var operationalLayers: [Layer] = []
+    
+    /// An array for all the layers removed from the map.
+    @State private var removedLayers: [Layer] = []
+    
+    var body: some View {
+        List {
+            Section(
+                header: Text("Operational Layers"),
+                footer: Text("Tap and hold on a list item to drag and reorder the layers.")
+            ) {
+                ForEach(operationalLayers, id: \.id) { layer in
+                    HStack {
+                        Button {
+                            // Remove layer from map on minus button press.
+                            map.removeOperationalLayer(layer)
+                            removedLayers.append(layer)
+                            operationalLayers.removeAll(where: { $0.id == layer.id })
+                        } label: {
+                            Image(systemName: "minus.circle.fill")
+                                .foregroundColor(.red)
+                                .imageScale(.large)
+                        }
+                        Text(layer.name)
+                    }
+                }
+                .onMove { fromOffsets, toOffset in
+                    // Reorder the map's operational layers on list item move.
+                    operationalLayers.move(fromOffsets: fromOffsets, toOffset: toOffset)
+                    map.removeAllOperationalLayers()
+                    map.addOperationalLayers(operationalLayers)
+                }
```

Comment:
```suggestion
                }
                .onDelete { indexSet in
                        // delete layer
                    }
```
Should swipe to delete be supported like in the Objective-C Sample @yo1995?

Reply:
Ideally it should be supported, but I think Caleb mentioned there is a bit of difficulty to get it right. Besides, Apple's Settings > Control Center page doesn't have swipe to delete so I feel the current UI is fine.

Reply:
I can probably get it work. I though we were forgoing that functionality since we had the "-" delete button that did the same thing. Do you want me to try to add the swipe to delete Ting?

Reply:
Okay, I implemented swipe to delete in another sample like [this](https://github.com/Esri/arcgis-maps-sdk-swift-samples/blob/des12437/Create-load-report/Shared/Samples/Create%20load%20report/CreateLoadReportView.Model.swift#L220-L227).

Reply:
All good for me either to have it or not.

Reply:
I added the functionality. Should be good now.

Reply:
OK. Because it is a button, it is kind of hard to swipe-delete without triggering the single tap gesture. Destiny's example is using an HStack of texts, so it is a bit different. But not a big deal.